frontends/concrete-python/concrete/fhe/compilation/module.py

```diff
@@ -207,6 +223,10 @@ def decrypt(
             Optional[Union[int, np.ndarray, Tuple[Optional[Union[int, np.ndarray]], ...]]]:
                 decrypted result(s) of evaluation
         """
+
+        if self.configuration.simulate_encrypt_run_decrypt:
```

Comment:
by the way, is it expected we have two versions of encrypt, run and decrypt, one for modules and one for classical? should it be factorized? maybe create an issue if it's needed

Reply:
It's a separate discussion, could you talk with @aPere3 about it.

Reply:
We known there are several places where the code can be cleaned and/or refactorized, I think there are no need to create a specific issue for that.

Reply:
I would factorize it before we continue to dup and start to diverge between the two things

Reply:
https://github.com/zama-ai/concrete-internal/issues/742

Reply:
It would be very long to deduplicate everything.